README.md

```diff
@@ -1 +1,17 @@
 # TLA+ Foundation
+
+# 1. Goals of the Foundation
+The TLA+ Foundation (TLAF) is an independent, non-profit organization dedicated to fostering the adoption of the TLA+ specification language in industry, academia, and education, guided by the overall goal to advance the practice of mathematically thinking in software engineering.  The mission of TLAF is active support and funding of the development of the TLA+ tools to broaden their use by practitioners.  If not directly funding work, TLAF creates synergies whenever possible.
+
+The TLA+ Foundation is also responsible for the evolution of the TLA+ specification language that guarantees long-term consistency.
+
+# 2. The values of the Foundation
+How we pursue the goals of HF is just as important as what the goals are. These core principles will guide the actions of the Foundation:
+
+* **Open source**. The TLA+ foundation will embrace the open-source ethos wholeheartedly. Whenever TLAF develops or sponsors the development of tools and infrastructure, it will all be open source under a permissive license.  This is not limited to the end result but also applies to how work is organized and carried out; the community at large has to be able to participate at each step of the journey.
+ 
+* **Empowering the community**. While the TLAF raises resources to fund development work, it does not compete with volunteer efforts from industry or academia.  The main focus is to address the tragedy of the commons to enable the success of others. The TLAF will boost, celebrate, and coordinate the contributions of volunteers, not supplant or replace them.
+
+* **Transparent**. All formal decision-making will be publicly accessible. We also strive to share ideas early. This is to enable asynchronous communication and collaboration. Only certain categories of sensitive information (e.g., financial and matters concerning particular individuals) may be kept confidential.
+
+* **Open, friendly, and diverse**. `We strive to make sure that all lifeforms are welcome, that all can contribute, and that it can all be done in a way that empowers our community members.
```

Comment:
Trailing \` here. The "lifeforms" thing strikes me as odd.

Reply:
I like the spirit of this, but I think the phrasing could be a bit better… just not sure what exactly that would look like yet